internal/util/argutil/argutil.go

```diff
@@ -85,3 +88,15 @@ func ParseFieldPath(path string) ([]string, error) {
 	}
 	return newParts, nil
 }
+
+// ResolveSymlink returns the resolved symlink path for the input path
+func ResolveSymlink(ctx context.Context, path string) (string, error) {
+	rp, err := filepath.EvalSymlinks(path)
+	if err != nil {
+		return "", err
+	}
+	if rp != path {
+		fmt.Fprintf(printer.FromContextOrDie(ctx).ErrStream(), "[WARN] resolved symlink %q to %q, please note that the symlinks within the package are ignored\n", path, rp)
```

Comment:
Can you pl. paste the CLI output for a few commands especially `fn render` and `fn eval` on a symlinks in the description of the PR to see how the output looks for the end-user.